src/styles/page-style.scss

```
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Only `box-sizing` property allows for `*` selector
```suggestion
  box-sizing: border-box;
```

---

@loralevitska, thanks a lot.